### DISCUSSION TITLE
can not use import { Ionicons } from '@expo/vector-icons';

### DISCUSSION THREAD

**Discussion by becodeicon** [community/community#139220] [Programming Help]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hi, 

https://github.com/Pushkal-vashishtha/BusinessDirectory-App-/tree/main

I have created new React Native project but I can not use Ionions icons and the basic command in follow not working;

import { Ionicons } from '@expo/vector-icons';

I have tried a lot of available solutions online but none of them worked.

I need any help or comment 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Sin-cy** [comment]
Hey if you haven't already, try this to import font `import * as Font from 'expo-font';` it could be a font loading issue. 
 
Im guessing you already tried reinstalling expo and it didn't work still.
Deleting your package-lock.json and it still also didn't work.

Final guess
It might be something to do with the Icon Size specified as I checked ur ts config and .json and I dont see the issue with the imports or the versions. Probably something to do with the size of those icons, or the view size or something. The Icons maybe too big and are just squashing each other out in that small box. Maybe the icons are just not in the right width and height or centered correctly.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @becodeicon , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
